frontend/ui/lib/widgets/default_app_bar.dart

```diff
@@ -1,10 +1,17 @@
 import 'package:flutter/material.dart';
+import 'package:tripmeout/router/router.dart';
+import 'package:tripmeout/widgets/routing_button_widget.dart';
 
 AppBar defaultAppBar(BuildContext context) {
   return AppBar(
     // TODO: Customize me yo.
-    title: Text('TripMeOut'),
+    title: Row(children: [Icon(Icons.airplanemode_active), Text('TripMeOut')]),
     // Don't show app bar back button, it looks gross.
-    leading: Container(),
+    actions: [
+      RoutingButton(
```

Comment:
nit: I don't think you need a creation link from the app bar...

Reply:
I was thinking coming from the view trip page. Should I revert this in another pr?